interactive _old.py

```diff
@@ -0,0 +1,174 @@
+import pygame
```

Comment:
Delete old code or testing code that are not part of your final submission in your repository.